webpack/dev.config.js

```diff
@@ -0,0 +1,27 @@
+const path = require('path');
+const webpack = require('webpack');
+const merge = require('webpack-merge');
+
+const baseConfig = require('./base.config.js');
+
+module.exports = merge(baseConfig, {
+  devtool: 'inline-source-map',
+  devServer: {
+    contentBase: './ui-src',
+    historyApiFallback: true,
+    inline: true,
+    host: '0.0.0.0',
+    port: 3000,
+    hot: true,
+    proxy: {
+      '/api': 'http://192.168.10.232:8080',
```

Comment:
Is there some way to make this configurable, without us having to modify the checked-in file?